_events/2022-06-27-master-defense-danyao-wang.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: event
+title: "Master Thesis Defense on Developer-Centric Test Amplification: User-Guided Test Amplification"
+categories: [events, defenses]
+start: "12:30"
+end: "14:30"
+speaker: Danyao Wang
+where: Hybrid
```

Comment:
Could you already add the room here or in the description below?

The zoom link you can then share in your Mattermost message (use the same one as we always do - I already sent that one to Ujwal 🙂)

Reply:
Yes, I added the address. Thanks for your suggestions!